src/resolvers/UserTag/organization.ts

```diff
@@ -0,0 +1,16 @@
+import { UserTagResolvers } from "../../types/generatedGraphQLTypes";
+import { OrganizationTagUser } from "../../models";
+
+export const organization: UserTagResolvers["organization"] = async (
+  parent
+) => {
+  const organizationObject = await OrganizationTagUser.findOne({
```

Comment:
This implementation is incorrect. You have the `organizationId` field available on `parent`. Use it to directly fetch the organization object from database.

Reply:
![image](https://user-images.githubusercontent.com/96648934/225642333-afb9eeca-c168-42cc-97db-9508d2fef366.png)
Using the same gives the following typescript error, which I think so is correct as parent here is `UserTag type` and not the `OrganizationTagUser model`, which does not have any `organizationId` field, but rather only an `organization` field which needs to be resolved. Or am I mistaken?

Reply:
@EshaanAgg You haven't mapped `UserTag` in graphql schema to `OrganizationUserTag` interface. Do that and typescript won't complain.

Reply:
@xoldyckk Done! Worked like a charm.